src/analysis/exploration/biomassPrecursorCheck.m

```diff
@@ -1,7 +1,11 @@
-function [missingMets, presentMets,coupledMets] = biomassPrecursorCheck(model,checkCoupling)
+function varargout = biomassPrecursorCheck(model, checkCoupling, checkConservedQuantities)

```

Comment:
May you change here to this format:
```
[missingMets, presentMets, coupledMets, missingCofs, presentCofs] = ...
```
in order to be consistent with all other functions?

Reply:
Well... with varargout, you could (potentially) allow getting only `missingCofs` and `presentCofs` but not `coupledMets`.  Which you can't if you have fixed values. his was actually a request by me. 
The problem for me is, that you have two different things that you might want to check, and I would love to keep them separate. But yes, this makes the function inconsistent with other functions :(